src/data/ad.json

```diff
@@ -1,6 +1,6 @@
 {
-  "title": "OpenWhisk and resin.io for edge computing",
-  "text": "A pair of writeups for using resin.io with OpenWhisk as a distributed compute service",
-  "link": "https://forums.resin.io/t/openwhisk-writeup/593",
-  "image": "https://cdn-images-1.medium.com/max/1200/1*EEJjDyoHUnuY90zm2d3Uqw.png"
+  "title": "Changes in resin.io support",
+  "text": "Our goal is for the entire resin.io community to become more knowledgeable with each issue solved and question answered.",
```

Comment:
@alisondavis17 I've added all of that in, and it looks pretty big for a preview, but not crazy. In the current formatting all the text is a link to the post, so can't really add extra links.

I'm guessing if we wanna include that link, it should also rather be in that post itself. It was, but not explicitly. I've added an expanded note there, let me know if it's okay:

>  (If you’re not a paid user but would like to become one, check our [support plans](https://resin.io/support) or contact us at sales@resin.io). 

What do you think?